server/views/formPages/incident/useOfForceDetails.html

```diff
@@ -103,8 +103,11 @@ <h1 class="govuk-heading-xl mainHeading">{{ pageTitle }}</h1>
     }
     )}}
 
-    <!-- Q6 -->
     {% if featureFlagDisplayDogAndTaserQuestions %}
+      <!-- Q6 -->
+      {% include "./partials/taser.njk" %}
+
```

Comment:
The format of  the taser questions is not used elsewhere in UoF. Therefore rather than a reusable macro, I've created a simpler view specifically for taser